protocol/vdp_protocol.yaml

```diff
@@ -0,0 +1,166 @@
+---
+"$schema": http://json-schema.org/draft-07/schema#
+"$id": https://github.com/instill-ai/vdp/blob/main/protocol/vdp_protocol.yaml
+title: VDP_Protocol
+type: object
+description: VDP Protocol structures
+properties:
+  batch_outputs:
+    "$ref": "#/definitions/BatchOutputs"
+definitions:
+  BatchOutputs:
+    type: array
+    items:
+      "$ref": "#/definitions/TaskOutput"
+  TaskOutput:
+    type: object
+    additionalProperties: true
+    minProperties: 2
```

Comment:
@xiaofei-du `BatchOutputs` here corresponds to a single model. Then it will be `oneof` classification, detection, keypoint, unspecified. minProperties=2 it mean that output could be combination of classification, detection, keypoint, unspecified. I just think it could be make confuse for use. What do you think?

Reply:
@Phelan164 , please check the latest commit.